scripts/checkpatch.pl

```diff
@@ -592,6 +592,20 @@ sub hash_show_words {
 	["__ATTR", 2],
 );
 
+our $api_defines = qr{(?x:
+	_POSIX_SOURCE|
+	_POSIX_C_SOURCE|
+	_XOPEN_SOURCE|
+	_XOPEN_SOURCE_EXTENDED|
+	_ISOC99_SOURCE|
+	_ISOC11_SOURCE|
+	_ATFILE_SOURCE|
+	_GNU_SOURCE|
+	_BSD_SOURCE|
+	_SVID_SOURCE|
+	_DEFAULT_SOURCE
```

Comment:
Non-blocking - prefer you alphabetize these.

Reply:
Good call. done.